drivers/generic/generic.go

```diff
@@ -16,17 +18,15 @@ import (
 
 type Driver struct {
 	*drivers.BaseDriver
-	SSHKey string
```

Comment:
Good catch @nathanleclaire 
Shouldn't we have a test to protect us from this kind of issue?